src/main/g8/build.sbt

```diff
@@ -9,7 +9,7 @@ lazy val commonSettings = Defaults.coreDefaultSettings ++ Seq(
   organization          := "$organization$",
   // Semantic versioning http://semver.org/
   version               := "0.1.0-SNAPSHOT",
-  scalaVersion          := "2.11.12",
```

Comment:
There were some issues with serialization etc in `2.12`, might be safer to stay on `2.11`

Reply:
@ravwojdyla sorry for late reply, hmm yeah, I haven't seen any of those for awhile now, that's why I figure we could push it. But if you wanna keep it let me know.

Reply:
I would prefer staying on `2.11`, and leave it up to people to change if they want.

Reply:
@ravwojdyla ok 😄